src/h5web/toolbar/controls/ScaleSelector/ScaleOption.tsx

```diff
@@ -1,15 +1,16 @@
 import type { ScaleType } from '../../../vis-packs/core/models';
-import styles from '../../Toolbar.module.css';
 import { H5WEB_SCALES } from '../../../vis-packs/core/scales';
+import styles from './ScaleSelector.module.css';
 
 function ScaleOption(props: { option: ScaleType }) {
   const { option } = props;
   const { Icon, label } = H5WEB_SCALES[option];
+
   return (
-    <>
+    <div className={styles.option}>
       <Icon className={styles.icon} />
-      {label}
-    </>
+      <span>{label}</span>
```

Comment:
The `span` brings back the margin of the `icon` (because of `.icon:last-child` in `Toolbar.module.css`).